ext/sqlite3/sqlite3.c

```diff
@@ -1248,7 +1248,7 @@ PHP_METHOD(SQLite3, openBlob)
 	}
 
 	if (ZEND_NUM_ARGS() >= 4 && CHECK_NULL_PATH(dbname, dbname_len)) {
-		zend_argument_type_error(4, "must not contain null bytes");
+		zend_argument_value_error(4, "must not contain any null bytes");
 		RETURN_THROWS();
 	}
```

Comment:
For the ones in sqlite3 it would be better to use de `p`/`P` ZPP format as it will handle null bytes automatically (wasn't used before due to the error message mentioning "path")

Reply:
Ah, that is convenient. Thanks!

Reply:
Used `p` in latest version of this PR